internal/extproc/translator/openai_openai.go

```diff
@@ -14,12 +14,9 @@ import (
 	"github.com/envoyproxy/ai-gateway/internal/extproc/router"
 )
 
-// newOpenAIToOpenAITranslator implements [Factory] for OpenAI to OpenAI translation.
-func newOpenAIToOpenAITranslator(path string) (Translator, error) {
-	if path == "/v1/chat/completions" {
-		return &openAIToOpenAITranslatorV1ChatCompletion{}, nil
-	}
-	return nil, fmt.Errorf("unsupported path: %s", path)
+// NewOpenAIToOpenAITranslator implements [Factory] for OpenAI to OpenAI translation.
+func NewOpenAIToOpenAITranslator() Translator {
```

Comment:
let's call it more precisely 
```suggestion
func NewChatCompletionOpenAIToOpenAITranslator() Translator {
```

Reply:
Done